module1/homework/grayscaleImages/main.cpp

```diff
@@ -1,7 +1,8 @@
+#include "compression.hpp"
+
 #include <array>
 #include <forward_list>
-
-// TODO: include
+#include <cstdint>
```

Comment:
You don't need to include cstdint here.You included this in header,which you are including here.